frontend/src/app/Settings/Facility/FacilityForm.test.tsx

```diff
@@ -110,13 +110,13 @@ describe("FacilityForm", () => {
   const renderWithUser = (facility = validFacility) => ({
     user: userEvent.setup(),
     ...render(
-      <MemoryRouter>
```

Comment:
All our tests that use the old `<MemoryRouter />` component now need to be refactored to use the new data-enabled version. I created a few helper functions to handle the boilerplate for all our tests.